core/src/main/java/com/sequenceiq/cloudbreak/service/audit/AuditEventService.java

```diff
@@ -28,27 +31,30 @@
     @Inject
     private StructuredEventRepository structuredEventRepository;
 
+    @Inject
+    private OrganizationService organizationService;
+
     public AuditEvent getAuditEvent(Long auditId) {
-        Optional<StructuredEventEntity> event = structuredEventRepository.findById(auditId);
-        return event.isPresent() ? conversionService.convert(event, AuditEvent.class) : null;
+        return getAuditEventByOrgId(organizationService.getDefaultOrganizationForCurrentUser().getId(), auditId);
     }
 
     public AuditEvent getAuditEventByOrgId(Long organizationId, Long auditId) {
-        StructuredEventEntity event = structuredEventRepository.findByOrgIdAndId(organizationId, auditId);
-        return event != null ? conversionService.convert(event, AuditEvent.class) : null;
+        StructuredEventEntity event = Optional.ofNullable(structuredEventRepository.findByOrgIdAndId(organizationId, auditId))
+                .orElseThrow(notFound("Audit event for id", auditId));
```

Comment:
the first parameter of the 'notFound' is the name of the resource and not an error message. It will create a message like 'Stack 5 not found.'

Reply:
I thought that the "Audit event for id '5' not found" would be more readable.

Reply:
Everywhere else it's the name of the resource, I think for the sake of consistency it's worth to change to AuditEvent or StructuredEvent.

Reply:
You're right, I change it then (and add some unit test for these functions).